friends/src/components/CreateFriend.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { axiosWithAuth } from '../utils/axiosWithAuth';
+
+class CreateFriend extends React.Component {
+    state = {
+      createFriend: {
+        name: '',
+        age: '',
+        email: ''
+      }
+    };
+  
+    handleChange = e => {
+      this.setState({
+        createFriend: {
+          ...this.state.createFriend,
+          [e.target.name]: e.target.value
+        }
+      });
+    };
+  
+    addFriend = e => {
+      e.preventDefault();
+      axiosWithAuth()
+        .post('/friends', this.state.createFriend)
+        .then(res => {
+                console.log(res);
```

Comment:
just for UI best practices, I would have put props.history.push('/protected') here so users don't have to click the link to see if friends have been added :)